apps/rush-lib/src/cli/logic/InstallManager.ts

```diff
@@ -34,6 +34,19 @@ const MAX_INSTALL_ATTEMPTS: number = 5;
 
 const wrap: (textToWrap: string) => string = wordwrap.soft(Utilities.getConsoleWidth());
 
+/**
+ * The "noMtime" flag is new in tar@4.4.1 and not available yet for \@types/tar.
+ * As a temporary workaround, augment the type.
+ */
+import { CreateOptions } from 'tar';
+export interface CreateOptions { // tslint:disable-line:interface-name
```

Comment:
>CreateOptions [](start = 17, length = 13)

Does this somehow append to the definitions coming from the @types package?

Reply:
Yes, it is [declaration merging](https://www.typescriptlang.org/docs/handbook/declaration-merging.html)

---
In reply to: [178998630](https://github.com/Microsoft/web-build-tools/pull/595#discussion_r178998630) [](ancestors = 178998630)